### PR TITLE
feat(Select): add error property

### LINKED ISSUE #154: Improve Select styles in case of using "clear" view
Select has the same styles for `focused` and `active/opened` states at present. This is confuse our users a little bit.
<details>
<summary>Issue</summary>

select-focus-demo

</details>

### PR DESCRIPTION
**Pull request by korvin89**
Closes #154

### CODE REVIEW

**Review by jhoncool — APPROVED** [review]

File: src/components/Select/components/SelectControl/SelectControl.scss [hunk 1]
```diff
@@ -1,93 +1,27 @@
...
 #{$block} {
-    @include mixins.button-reset();
-    @include button-text-color(var(--yc-color-text-primary));
-    @include mixins.pin($block, $block, var(--yc-select-border-radius));
-
-    box-sizing: border-box;
-    display: inline-flex;
-    align-items: center;
     width: 100%;
     min-width: 34px;
```
Comment (ValeraS) [inline src/components/Select/components/SelectControl/SelectControl.scss:9]: is it needed?

File: src/components/Select/components/SelectControl/SelectControl.scss [hunk 2]
```diff
@@ -1,93 +1,27 @@
...
-    border: 1px solid var(--yc-color-line-generic);
```
Comment (ValeraS) [inline src/components/Select/components/SelectControl/SelectControl.scss:30]: let's add `text-align: right` to place icon correctly.

image

**Review by ValeraS — CHANGES_REQUESTED** [review]

File: src/components/Select/utils.tsx [hunk 3]
```diff
@@ -247,3 +248,15 @@ export const getFilteredFlattenOptions = (args: {
         return acc;
     }, [] as FlattenOption[]);
 };
+
+export const mapToButtonView = (view: NonNullable<SelectProps['view']>): ButtonView => {
+    switch (view) {
+        case 'clear': {
+            return 'flat';
+        }
+        case 'normal':
```
[the remark this answers is not in the document]
Comment (korvin89) [reply src/components/Select/utils.tsx]: fixed

File: src/components/Select/components/SelectControl/SelectControl.scss (hunk 1 above)
Comment (korvin89) [reply src/components/Select/components/SelectControl/SelectControl.scss:9]: `width: 100%` - to fill all available place of control`s wrapper
`min-width: 34px` - useless style

**Review by ValeraS — APPROVED** [review]

**Pull request merged**